src/action.js

```diff
@@ -6,59 +6,62 @@ const registerDependencyAndroid = require('./android/registerNativeModule');
 const registerDependencyIOS = require('./ios/registerNativeModule');
 const copyAssetsAndroid = require('./android/copyAssets');
 const copyAssetsIOS = require('./ios/copyAssets');
+const pjson = require(path.join(process.cwd(), './package.json'));
 
+/**
+ * Namespace log messages
+ */
 log.heading = 'rnpm-link';
 
 /**
  * Returns an array of dependencies that should be linked/checked.
+ * @param {Object} pconfig Project config
  */
-const getProjectDependencies = () => {
-  const pjson = require(path.join(process.cwd(), './package.json'));
```

Comment:
That was intended. Becase the check on line 31 would've returned false if there was no package.json. Otherwise, we get require error from the very first moment. I am the fun of the previous version.